labs/firepower-restapi-101/2.md

```diff
@@ -1,67 +1,56 @@
-
-The first time you connect to the API Explorer you may receive an error that the connection is not secure due to an invalid certificate. You will need to add an exception in your browser to use the certificate and accept the connection.
 ![Figure: FMC REST API Explorer screen shot ](/posts/files/firepower-restapi-101/assets/images/fmcapiexp.PNG)
 
 ## Authentication to the API Explorer
-The REST API relies on the same authentication as the FMC UI. Each function in the REST API maps to permissions in the FMC.
-
-You can log into the API Explorer using any account on the FMC, but you will only be able to perform the functions for which the account has permissions. For more information on setting permissions for user roles, see the [Firepower Management Center Configuration Guide](http://www.cisco.com/c/en/us/td/docs/security/firepower/610/configuration/guide/fpmc-config-guide-v61/logging_into_firepower_system.html).
 
-FMC REST API uses token-based authentication. To invoke a REST call on FMC, you need to use a valid access token. On every REST call to access resource URLs you need to add a header with "key" set to “X-auth-access-token” and “value” containing a valid access token. API Explorer also follows this, but it is automated by Explorer UI. You can learn more in [Firepower REST API 102 lab](https://learninglabs.cisco.com/lab/firepower-restapi-102/step/1).
+FMC REST API uses token-based authentication. You need a valid access token to invoke a REST call, and every REST call must include a header in which `key` is set to `X-auth-access-token` and the `value` contains the access token. You can learn more in [Firepower REST API 102 lab](https://learninglabs.cisco.com/lab/firepower-restapi-102/step/1).
 
-## Logging into API Explorer
-The REST API uses the same authentication and permission model as the Firepower Management Center. See the [Firepower Management Center Configuration Guide](http://www.cisco.com/c/en/us/td/docs/security/firepower/610/configuration/guide/fpmc-config-guide-v61/logging_into_firepower_system.html) for more information.
+The API Explorer automates the access token process, and you can log into the API Explorer using any FMC account, but you will only be able to perform the functions for which the account has permissions. For more information on setting permissions for user roles, see the [Firepower Management Center Configuration Guide](http://www.cisco.com/c/en/us/td/docs/security/firepower/610/configuration/guide/fpmc-config-guide-v61/logging_into_firepower_system.html).
 
+## Logging into FMC
 
-Make sure you have an account on your Firepower Management Center with the required permissions to perform the tasks you want to perform using the REST API.
+Detailed information about logging into FMC is documented in the [Firepower Management Center Configuration Guide](http://www.cisco.com/c/en/us/td/docs/security/firepower/610/configuration/guide/fpmc-config-guide-v61/logging_into_firepower_system.html).
 
```

Comment:
Point to 6.2.3 instead of 6.1 - https://www.cisco.com/c/en/us/td/docs/security/firepower/623/configuration/guide/fpmc-config-guide-v623.html -- unless you find out the always-on Sandbox is 6.1. Probably want to match to the always-on Sandbox version.